src/n64sys.c

```diff
@@ -26,6 +28,8 @@
  * @{
  */
 
+extern volatile void __mem_start;
```

Comment:
If you decide to keep it, please add a comment here that says that it's defined in `n64.ld` as `0x80000000`, so at least it's easier for readers.

Also I don't think it has to be `volatile`.

Reply:
Sure I will do. Better to cast it to volatile later (cache invalidate funcs expect a volatile param)